reana_client/cli/utils.py

```diff
@@ -31,7 +31,7 @@ def wrapper(*args, **kwargs):
     return wrapper
 
 
-def access_token_check(ctx, access_token):
+def access_token_check(ctx, param, access_token):
```

Comment:
I think that using `param` as a parameter name could be confusing along with `access_token`. What do you think of either renaming to `def access_token_check(ctx, param, value):` (so it is clear that one is the param name and the other the value, more generic) or ignore the `param` parameter `def access_token_check(ctx, _, access_token):` (more specific)?